server/handlers/cdn-caching.js

```diff
@@ -1,3 +1,33 @@
+/**
+ * How quintype handles toggle between different CDN providers
+ * - For now we have support for `Cloudflare` and `akamai`.
+ * - The default CDN provider is set as `Cloudflare`.
+ *
+ *   How to toggle between cdn providers:
+ *   - If your app is cloned from malibu then, in `malibu/app/server/app.js` pass `cdnProvider` with required value to `isomorphicRoutes`(No need to pass if `cloudflare` is your cdn provider).
+ *   Note: Contact Quintype before switching the CDN.
+ *
+ *   ```javascript
+ *   import { isomorphicRoutes } from "@quintype/framework/server/routes";
+ *   isomorphicRoutes(app, {
+ *   ...
+ *   ...
+ *   ...
+ *   cdnProvider: "YOUR_CDN_PROVIDER_NAME",
+ *   });
+ *   ```
+ *   - In `quintype-node-framework`, the functionality to set the required response headers is written in file: `server/handlers/cdn-caching.js`
+ *   - If the cdn provider is `akamai`, then we are updating the response headers as below
+ *   ```javascript
+ *   res.setHeader("Edge-Control", "private,no-cache,no-store,max-age=0");
+ *   res.setHeader("Edge-Control",`public,maxage=${sMaxAge},stale-while-revalidate=1000,stale-if-error=14400`);
+ *   res.setHeader("Edge-Cache-Tag", _(cacheKeys).uniq().join(","));
+ *   res.setHeader("Edge-Control","public,maxage=60,stale-while-revalidate=150,stale-if-error=3600");
```

Comment:
If someone is not using our fwk, like esakal, we should give examples of setting CF and Fastly headers, Fastly may be used as backup CDN